src/test/java/uk/gov/digital/ho/pttg/SmokeTest.java

```diff
@@ -1,11 +1,63 @@
 package uk.gov.digital.ho.pttg;
 
+import com.google.gson.*;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.Before;
 import org.junit.Test;
+import uk.gov.digital.ho.pttg.api.FinancialStatusRequest;
+import uk.gov.digital.ho.pttg.api.HttpResponse;
+
+import java.lang.reflect.Type;
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.fail;
 
 public class SmokeTest {
 
+    private final String IP_API_PATH = "/incomeproving/v3/individual/financialstatus";
+
+    private String ipApiRootUrl;
+    private CloseableHttpClient httpClient;
+    private SimpleHttpClient simpleHttpClient;
+    private Gson gson;
+
+    @Before
+    public void setUp() {
+        ipApiRootUrl = getMandatoryEnvVar("IP_API_ROOT_URL");
+        simpleHttpClient = new SimpleHttpClient();
+        gson = new GsonBuilder().registerTypeAdapter(LocalDate.class, new LocalDateSerializer())
+                                .create();
+    }
+
     @Test
-    public void testStartup() {
-        // TODO EE-6825 Tests yet to be added - for now it is sufficient to check that the tests run
+    public void testIpApi() {
+        String body = buildRequest();
+        System.out.println(body);
```

Comment:
We don't generally need to output diagnostics in tests.

Reply:
Agreed. I accidentally left those from when I was developing it.